hardhat/scripts/09_deployAndConfig.js

```diff
@@ -0,0 +1,94 @@
+const { execSync, spawn } = require("child_process");
```

Comment:
spawn imported but unused

Reply:
Yes, it's used in the commented code below. The commented code is better cause it gives a fluid output but have issue cause it adds empty lines and breaks the display. 

Not solved yet